website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -1,12 +1,10 @@
 ---
 title: Quickstart for Helm-based Operators
 linkTitle: Quickstart
-weight: 100
-description: A simple set of instructions that demonstrates the basics of setting up and running a Helm-based operator.
+weight: 1
+description: This guide walks through an example that demonstrates the basics of setting up and running a Helm-based operator powered by [Helm][helm-official]  using tools and libraries provided by the Operator SDK
 ---
 
-This guide walks through an example of building a simple nginx-operator powered by [Helm][helm-official] using tools and libraries provided by the Operator SDK.
-
```

Comment:
Revert this. I _just_ changed it to what's currently there.

Reply:
It shows the info duplicated which is very strange